server/test/test_diffexp.py

```diff
@@ -88,16 +88,34 @@ def test_cxg_generic(self):
     def test_cxg_sparse(self):
         with tempfile.TemporaryDirectory() as dirname:
             sparsename = os.path.join(dirname, "sparse.cxg")
+            sparse_col_shift_name = os.path.join(dirname, "sparse_col_shift.cxg")
             densename = os.path.join(dirname, "dense.cxg")
             source_h5ad = anndata.read_h5ad(f"{PROJECT_ROOT}/example-dataset/pbmc3k.h5ad")
+
             # create a cxg sparse array
+            # TODO:  pbmc3k.h5ad is 100% non-zero.  Converting it to sparse is useful, but it would
```

Comment:
I suggest creating a couple of synthetic H5AD files to test the various encoding schemes, and include it with this PR so we have better test coverage.

Reply:
Sounds good, added.